Lib/test/test_exceptions.py

```diff
@@ -1285,6 +1285,18 @@ def g():
                 next(i)
                 next(i)
 
+    def test_assert_shadowing(self):
+        # Shadowing AssertionError would cause the assert statement to
+        # misbehave.
+        globals()['AssertionError'] = TypeError
```

Comment:
Why use `globals()['AssertionError']` instead of just setting `AssertionError`?

Reply:
The bug involves `AssertionError` being shadowed at the global level, no?

Reply:
I meant that you can write
```python
global AssertionError
AssertionError = TypeError
```

Reply:
Okay. I will make that change.